api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 7.0
 
+**Feature level 168**
+
+* [`PATCH /users/me/subscriptions/muted_topics`](/api/mute-topic):
+  No error is raised in the case of requests to mute an already
+  muted topic. It is an easy success.
+  Previously, the error `Topic is already muted` was raised.
```

Comment:
We try to keep these brief and comprehensible without knowing anything about the endpoint or having specific strings. So I'd write "Trying to mute a topic that is already muted now results in a success response, rather than an error."